File: Bee 1020 - Age in Days/AgeInDays.py
#Bee 1020 - Age in Days 

dias = int(input())
anos = (dias/365)
meses = ((dias%365)/30)
dias = ((dias%365)%30)

print("{} ano(s)\n{} mes(es)\n{} dia(s)".format(int(anos), int(meses), int(dias)))

File: Bee 1020 - Age in Days/AgeInDays.cpp
//Bee 1020 - Age in Days 

#include <iostream>
#include <iomanip>
using namespace std;
 
int main() {
    int diasIn, diasOut, meses, anos;
    cin >> diasIn;    
    
    anos = (diasIn/365);
    meses = ((diasIn%365)/30);
    diasOut = ((diasIn%365)%30);

    cout << anos << " ano(s)" << endl << meses << " mes(es)" << endl << diasOut << " dia(s)" << endl;
    
    return 0;
}